webapp/app/css/treeherder.css

```diff
@@ -11,8 +11,8 @@ body {
     line-height: inherit; /* override bootstrap */
 }
 
-a:visited {
-    color: #a842ca;
+a:visited:not(.show-hide-more) {
```

Comment:
I think I'd prefer avoiding the browser seeing show/hide more as a visited link in the first place - eg by replacing show/hide more with a span (in another PR, don't worry about it here), since <a> is not really appropriate, since it's not really a link. ie: http://stackoverflow.com/a/135248

@camd - thoughts?

Thank you for the PR though - much better colour :-)